### ISSUE TITLE
Couple of questions about ACF-like functionality

### ISSUE THREAD

**Issue by iamntz** [WebDevStudios/CMB2#403]
Hi guys. I found this plugin a while back, but until now I didn't tried. [This comparison](https://docs.google.com/spreadsheet/ccc?key=0AoY8IFUX301qdFhBaERLUEUwa3U0YjFYTnBmaU1mbmc&usp=drive_web#gid=3) table though made me to give it a try.

Because I'm coming from ACF world, I'm looking for two functionalities that I'm not sure if they are included or not.

The first is this: _Flexible Repeatable Fields (like Loop, but can choose to add different defined sets of fields per row)_ (copied from the document above, which is marked as an existing feature).

Long story short, this looks like a repeatable field, but when you hit „Add another entry” you can pick various entry types.

The second feature I'm looking for is JS conditionals. That is showing/hiding various stuff based on various states (e.g. hide datepicker if a checkbox is checked, show map if a field is filled and so on).

That's all :)

Thanks!

**Comment by baljindersingh88** [comment]
The sheet looks informative.

**Comment by sc0ttkclark** [comment]
Yeah, someone did their homework!

**Comment by baljindersingh88** [comment]
Let me know if i can be of any help

**Comment by sc0ttkclark** [comment]
The Flexible Repeatable Fields isn't available inside of CMB2 natively yet, I'm not sure if someone has built something for it. There's no JS conditionals either, but I could be wrong there.

It does have loop fields / repeatable fields though, and lets you repeat groups of fields. I'm looking into improving "group" fields so they can handle groups inside of groups, multi-level.

**Comment by baljindersingh88** [comment]
Flexible Repeatable Fields are available but not sure about JS conditions.That will be a good contribution.

**Comment by sc0ttkclark** [comment]
I wasn't able to see this exact functionality available from CMB2 -- http://www.advancedcustomfields.com/resources/flexible-content/ -- there aren't Layouts or context-specific groups of fields in that way. You could add multiple groups of fields though, and us JS conditionals manually.

**Comment by baljindersingh88** [comment]
I think you are correct.This exact functionality is not available in CMB2.

**Comment by iamntz** [comment]
Meanwhile i found this: https://github.com/WebDevStudios/WDS-Simple-Page-Builder

Although it does some parts that i also need on current project, it doesn't play nice as a flexible content replacement :(

Oh well...

**Comment by baljindersingh88** [comment]
Yes this can he helpful.It's not flexible as ACf but still will be helpfull.
Scott talk to you later.Have to work on some functionality.The deadline is
approcahing.It's 12:00 PM  here in my office.
Bye
